relay-server/src/envelope.rs

```diff
@@ -382,6 +385,7 @@ impl fmt::Display for AttachmentType {
             AttachmentType::Breadcrumbs => write!(f, "event.breadcrumbs"),
             AttachmentType::UnrealContext => write!(f, "unreal.context"),
             AttachmentType::UnrealLogs => write!(f, "unreal.logs"),
+            AttachmentType::ViewHierarchy => write!(f, "event.viewhierarchy"),
```

Comment:
For the name, I'm leaning towards `"event.view_hierarchy"` but not strongly opinionated for the same reason you describe in the PR description. I'd match it with the type coming in the envelope though.

Reply:
This was already discussed here: https://github.com/getsentry/rfcs/pull/33#discussion_r1032075538. But there seems to be a strong preference for `view_hierarchy` so go for it.

Reply:
Ah I knew it was mentioned somewhere, but couldn't find it - thanks for linking!